HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -310,6 +310,47 @@ code::
 "mississippi".split("i").postcs;
 ::
 
+method::insertAll
+Return a new collection of the same kind, with all elements of another collection or object inserted, starting from a given index.
+
+argument::index
+The index at which to start insertion. If too large or too small, append or prepend the object's items.
+
+argument::obj
+The obj to insert (usually a collection, but anything that responds to code::++::).
+
+discussion::
+code::
+a = [1, 2, 3, 4, 5, 6];
+b = [100, 200];
+a.insertAll(2, b); // [ 1, 2, 100, 200, 3, 4, 5, 6 ]
```

Comment:
I'd like to have here something like

```
a = [10, 20, 30, 40, 50, 60];
b = [100, 200];
a.insertAll(2, b); // [ 1, 2, 100, 200, 3, 4, 5, 6 ]
```

to prevent (reader's) mixup of indices and values.

or maybe even

```
a = [\l, \m, \n, \o, \p, \r];
b = [\X, \Y];
a.insertAll(2, b); // [ \l, \m, \X, \Y, \n, \o, \p, \r ]
```